pkg/kuttlctl/cmd/test.go

```diff
@@ -58,6 +58,7 @@ func newTestCmd() *cobra.Command {
 	mockControllerFile := ""
 	timeout := 30
 	reportFormat := ""
+	reportName := "kuttl-report"
```

Comment:
These vars are set here, but it seems like the main value of setting the default value here is as a reminder of what the default is, since they get overwritten by the config file, yeah?

Reply:
after some refactoring, I understand better your question.  IMO we should refactor the init to the be the order defined and not "check" the value throughout code.